File: app/src/main/java/space/pal/sig/model/dto/FactDto.java
package space.pal.sig.model.dto;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;
import space.pal.sig.model.Fact;

@Setter
@Getter
public class FactDto {

    @SerializedName("name")
    private String name;

    public Fact toFact() {
        return Fact.builder().name(name).build();
    }
}

File: app/src/main/java/space/pal/sig/model/dto/FactDto.kt
package space.pal.sig.model.dto

import space.pal.sig.model.entity.Fact

/**
 * SpaceNow
 * Created by Catalin on 12/25/2020
 **/
data class FactDto(
        val content: String,
        val index: String
) {

    fun toFact(): Fact {
        return Fact(content)
    }

}
